projects/test-bases/common/golang-1.10.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: List
+items:
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: common-test-base-golang-1.10
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: common-test-base:golang-1.10
+    source:
+      type: Git
+      git:
+        ref: skuznets/image-registry-build
```

Comment:
eek, this is a mistake on my end -- can we make these refs `master`?

Reply:
Ah I missed that one, I've updated it now and the golang 1.9 config